gravity.sh

```diff
@@ -377,7 +376,6 @@ gravity_ParseFileIntoDomains() {
       if($0 ~ /^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+$/) { $0="" }
       if($0) { print $0 }
     }' "${source}" > "${destination}"
-    chmod 644 "${destination}"
```

Comment:
This line is probably still needed, as the destination could be created as part of this command.